package.json

```diff
@@ -21,14 +29,27 @@
   ],
   "author": "Google LLC",
   "license": "Apache-2.0",
-  "repository": "github:google/h2-auto-push",
+  "repository": "github:google/node-h2-auto-push",
+  "ava": {
+    "files": [
+      "build/test/*-test.js"
+    ]
+  },
+  "nyc": {
+    "exclude": [
+      "build/test"
+    ]
+  },
   "dependencies": {
     "bloomfilter": "0.0.16"
```

Comment:
This doesn't use a semver range. Intentional?

Reply:
Not intentional. Fixed.